modal-js/src/client.ts

```diff
@@ -13,6 +13,8 @@ import {
 import { ClientType, ModalClientDefinition } from "../proto/modal_proto/api";
 import { profile, Profile } from "./config";
 
+let modalAuthToken: string | undefined;
```

Comment:
Note that the way we propagate auth tokens is a little different in Go vs. Typescript.

In Typescript, we propagate auth tokens via auth middleware / a grpc interceptor in `client.go`.

In Go, we propagate auth tokens via context in `function.go`.

I'm open to changing Go's implementation to also propagate auth tokens via grpc interceptors for consistency if we think that's better, but it might be a bit trickier since Go is multithreaded?